config/config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "daemon_endpoint": "localhost:9000",
+  "daemon_macaroon": "string",
+  "kraken_ws_endpoint": "ws.kraken.com",
+  "markets": [
+    {
+      "base_asset": "lbtc",
```

Comment:
This should be Liquid asset hash. How this could work oterwise?

UpdateMarketPrice wants it like this.